web-services/query/src/main/java/datawave/webservice/query/logic/BaseQueryLogic.java

```diff
@@ -193,6 +195,16 @@ public Iterator<T> iterator() {
         return iterator;
     }
     
+    @Override
+    public QueryLogicTransformer getEnrichedTransformer(Query settings) {
```

Comment:
Is there a reason not to reuse getTransformer() instead of creating a new method? If an enriched transformer is configured we probably always want to use it?

Reply:
The reason was that the underlying transformer still needed to be accessible in various places.  Also the getTransformer method is overridden almost everywhere.  I want to ensure that the response transform shim is applied no matter what underlying transformer is returned.

Reply:
Now that I think about it, I should probably make this method final to ensure that nothing overrides it.

Reply:
FWIW, I had the same question as Charlie as well here. 

Consider renaming `getTransformer` and its usages/implementations to `getBaseTransformer`, documenting it as an internal API (maybe making it protected) and renaming `getEnrichedTransformer` to `getTransformer` and making it `public final`?